ln-simln-jamming/src/main.rs

```diff
@@ -348,6 +358,58 @@ fn check_reputation_status(cli: &Cli, status: &NetworkReputation) -> Result<(),
     Ok(())
 }
 
+// NOTE: just copied this from simln for now.
```

Comment:
I don't think it's a total tragedy if we have to duplicate some code - if there's a nice refactor we could upstream to simln let's do it, but this is a custom use case so it's reasonable enough if there isn't anything that suits our use.